snowflake/README.md

```diff
@@ -117,3 +141,84 @@ Snowflake credentials are required.
 
 We are in the process of adding support for applications to Snowflake-powered
 Theia, starting with NetworkPolicy recommendation.
+
+## Network flow visibility with Grafana
+
+We use Grafana as the tool to query data from Snowflake, and visualize the
+networking flows in the cluster(s).
+
+### Configure datasource
+
+Datasource and pre-built dashboards will be configured automatically in Grafana.
+To configure datasource, export the following environment variables:
+`SNOWFLAKE_WAREHOUSE`, `SNOWFLAKE_DATABASE`, `SNOWFLAKE_ROLE`(Optional).
+
+ Name                     | Description
+------------------------- | ------------
+ SNOWFLAKE_WAREHOUSE      | Specifies the virtual warehouse to use once connected.
```

Comment:
I think we should add the other 3 env vars to the table (`SNOWFLAKE_ACCOUNT`, `SNOWFLAKE_USER`, `SNOWFLAKE_PASSWORD`). No reason to treat them differently.

Reply:
done

I was thinking users have already exported the other three env vars in a previous step. But it makes more sense to list them in the table.